web/src/app/modules/views/main/editors/modules/tool-pallette/components/tool-pallette.component.html

```diff
@@ -1,11 +1,11 @@
 <form *ngIf="isVisible" class="form-inline tool-bar">
     <ng-container *ngFor="let tool of tools">
-        <button id="{{'toolbar-' + tool.name + '-button'}}" title="{{tool.name | translate}}" class="{{'btn btn-sm btn-outline-' + tool.color}}" [ngClass]="{'active': isActive(tool)}" (click)="onClick(tool, $event)">
+        <button id="{{'toolbar-' + tool.name + '-button'}}" title="{{tool.name | translate}}" class="{{'btn btn-sm btn-outline-' + tool.color}} my-1" [ngClass]="{'active': isActive(tool)}" (click)="onClick(tool, $event)">

```

Comment:
Where is this new css class "my-1" defined?

Reply:
It is provided in Bootstrap framework
m -> margin
y -> y-axis
1 -> value of the margin

Reply:
Ah, I see. Good!